src/hyperlane-adapter/interfaces/IBatchedMessageDispatcher.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
All these files are already on main, no? You should import them from their canonical place

Reply:
I have moved the interfaces apart from IBatchedMessageDispatcher and ISingleMessageDispatcher, because the ones I'm using for the hyperlane are payable whereas the one defined by pool together aren't

Reply:
@asselstine Would be curious to hear how to handle this case? Should we modify the existing interface to make it payable?